Freebsd ARM support

## Feature Request
Any chance of a Freebsd arm build for Telegraf?  With Netgate starting to release more ARM based PFSense devices, telegraf is the last feature I need to fully switch off x86.  

I know the Freebsd ports tree is normally pretty good, but there's currently no support for telegraf on ARM.  As telegraf is more than happy to run on Freebsd and Linux ARM, it seems like there's a build we're just missing.

Maybe this would be better to contact the maintainer of the telegraf builds on freebsd, but I'd trust influx more.  

### Proposal:
Currently, pfsense, and mainly freebsd, are missing a telegraf package on ARMv7.

### Current behavior:
No freebsd ARM package.

### Desired behavior:
A freebsd ARM package.  

### Use case: [Why is this important (helps with prioritizing requests)]
PFSense on ARM

I tried doing a quick build, but it's failing at the `gopsutil` plugin (output at end).

Both cross-compiling, and compiling directly on an armv7 FreeBSD machine, fail when building `gopsutil`.

Since telegraf depends on `gopsutil`, I recommend, @SomKen, that perhaps you can file an issue in @shirou's `shirou/gopsutil` project.
I'm afraid I can't perform the build, until there is full armv7 (FreeBSD) support from that dependency.

armv7 FreeBSD `gopsutil` build output:

`github.com/shirou/gopsutil/cpu`
`
../../shirou/gopsutil/cpu/cpu_freebsd.go:25:16: undefined: cpuTimes
../../shirou/gopsutil/cpu/cpu_freebsd.go:42:31: undefined: cpuTimes
../../shirou/gopsutil/cpu/cpu_freebsd.go:66:38: undefined: cpuTimes
../../shirou/gopsutil/cpu/cpu_freebsd.go:72:15: undefined: cpuTimes
../../shirou/gopsutil/cpu/cpu_freebsd.go:87:13: undefined: cpuTimes
`
`github.com/shirou/gopsutil/disk`
`
../../shirou/gopsutil/disk/disk_freebsd.go:26:36: undefined: MNT_WAIT
../../shirou/gopsutil/disk/disk_freebsd.go:31:15: undefined: Statfs
../../shirou/gopsutil/disk/disk_freebsd.go:32:28: undefined: MNT_WAIT
../../shirou/gopsutil/disk/disk_freebsd.go:38:17: undefined: MNT_RDONLY
../../shirou/gopsutil/disk/disk_freebsd.go:41:17: undefined: MNT_SYNCHRONOUS
../../shirou/gopsutil/disk/disk_freebsd.go:44:17: undefined: MNT_NOEXEC
../../shirou/gopsutil/disk/disk_freebsd.go:154:9: undefined: Bintime
../../shirou/gopsutil/disk/disk_freebsd.go:163:22: undefined: Statfs
../../shirou/gopsutil/disk/disk_freebsd.go:167:54: undefined: Statfs
../../shirou/gopsutil/disk/disk_freebsd.go:182:32: undefined: Devstat
../../shirou/gopsutil/disk/disk_freebsd.go:44:17: too many errors`

--- a/Gopkg.lock
+++ b/Gopkg.lock
@@ -973,7 +973,7 @@
   version = "v1.2.0"
 
 [[projects]]
-  digest = "1:02135a4151567d48ebff6cf36f73b5d8dee566855df84ffd96111d5225848bb7"
+  digest = "1:066c1020d667e25a0614b56aee1f9ac47e75c77de98eddfb51e9be02c68c1577"
   name = "github.com/shirou/gopsutil"
   packages = [
     "cpu",
@@ -986,8 +986,8 @@
     "process",
   ]
   pruneopts = ""
-  revision = "ccc1c1016bc5d10e803189ee43417c50cdde7f1b"
-  version = "v2.18.12"
+  revision = "071446942108a03a13cf0717275ad3bdbcb691b4"
+  version = "v2.19.01"
 
 [[projects]]
   branch = "master"

